### DISCUSSION TITLE
how to user private docker artifactory to deploy application

### DISCUSSION THREAD

**Discussion by akashkumar58-zz** [kubevela/kubevela#879] [Q&A]
- How can I use a private docker/container artifactory in the Appfile to deploy my application? I tried using imagePullSecret using which we can pull images from private artifactory in native K8S environment, but it does not seem to work. Could someone help me in this?

**Comment by zzxwill** [comment]
@hongchaodeng can help with the question.

**Comment by ryanzhang-oss** [comment]
There is no difference in KubeVela vs normal K8s. I wonder if you can paste the setup you use and the error you get when you use `imagePullSecret`?
